src/test/groovy/org/prebid/server/functional/tests/TopicsHeaderSpec.groovy

```diff
@@ -201,42 +197,35 @@ class TopicsHeaderSpec extends BaseSpec {
 
     def "PBS should overlap segments when Sec-Browsing-Topics header present in request"() {
         given: "Default basic BidRequest with generic bidder"
-        def requestSegment = PBSUtils.getRandomNumber().toString()
-        def modelVersion = PBSUtils.getRandomNumber()
-        def segmentId = PBSUtils.getRandomNumber().toString()
-
-        def bidRequest = BidRequest.defaultBidRequest.tap {
+        def randomSegClass = PBSUtils.randomNumber as String
+        def randomSegment = PBSUtils.randomNumber
+        def bidRequest = BidRequest.getDefaultBidRequest().tap {
             user = new User().tap {
                 data = [new Data().tap {
-                    name = "my_domain"
+                    name = PRIVACY_SENDBOX_DOMAIN
+                    segment = [new Segment(id: randomSegment)]
                     ext = new ExtData().tap {
                         segtax = 600
-                        segclass = modelVersion
+                        segclass = randomSegClass
                     }
                 }]
             }
         }
 
-        and: "Prepare valid Sec-Browsing-Topic value header"
-        def oneSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(
-                1,
-                [requestSegment],
-                modelVersion)
-        def headerSegmentId = PBSUtils.getRandomNumber().toString()
-        def twoSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(
-                1,
-                [headerSegmentId],
-                modelVersion)
+        and: "Prepare browsing topics header headers"
+        def oneSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(1, [PBSUtils.randomNumber as String], randomSegClass)
+        def secondSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(1, [PBSUtils.randomNumber as String], randomSegClass)
```

Comment:
`first` instead of `one,` but why are you using it in some cases one, two and in this one, `first`, `second`?